tests/performance_meter/test_performance.py

```diff
@@ -0,0 +1,221 @@
+import logging
+import asyncio
+import time
+import random
+import statistics
+import pytest
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.utilities import wait_until
+
+
+async def async_command(duthost, command):
+    return duthost.command(command)
+
+
+async def async_command_ignore_errors(duthost, command):
+    try:
+        return duthost.command(command, module_ignore_errors=True)
+    except Exception:
+        return
+
+
+# Defining an op.
```

Comment:
Can we define the op/success criteria in another files? Then user can easily extend it without reading unrelated code